package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "devDependencies": {
+    "markdown-link-check": "^3.7.3"
+  },
+  "scripts": {
+    "link-check": "find docs -name \\*.md -print0 | xargs -0 -n1 markdown-link-check"
```

Comment:
Minor nitpick: This should not be an NPM script — they're supposed to work everywhere and this won't work on Windows cmd.exe. We should move this in `circle.yml` and use it through `npx`

Reply:
Tests are failing because there are broken links. I think you might be overthinking this one it’s just a quicky to help doc writers out if they want to use it. 

@rowa97 @rossmcdonald are you both on OS X or is one of you on Windows? If so we can probably get away with this as is for now. And if you are on windows, you can find another script to check broken links, oooor you can just not run it manually and let circle tell you.

Reply:
We can leave it as it is, I do not really mind — I saw it and thought it was a good idea to point it out.